src/specs/components/media-container/demo/data--embed-video.js

```diff
@@ -1,6 +1,8 @@
 module.exports = {
+  sr_video_player: 'Video player',
+  title: 'New digital strategy',
   description:
     'The European Commission has put forward ambitious yet realistic proposals for a modern EU budget. It is time for an EU budget that reflects rapid developments in innovation, the economy, the environment and geopolitics, amongst others.',
   embedded_media:
-    '<iframe title="Video player" width="350" height="197" src="https://www.youtube.com/embed/fgi-GSCB6ho" frameBorder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowFullScreen></iframe>',
+    '<iframe title="New digital strategy - Video player" width="350" height="197" src="https://www.youtube.com/embed/fgi-GSCB6ho" frameBorder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowFullScreen></iframe>',
```

Comment:
why do we hardcode the title here? This way we are not really going to see this attribute added by the js..